ignite/contrib/handlers/lr_finder.py

```diff
@@ -193,7 +193,7 @@ def plot(
         skip_end: int = 5,
         log_lr: bool = True,
         display_suggestion: bool = True,
-        filepath: Optional[str] = None,
+        axes: Any = None,
```

Comment:
Let's call it `ax` as in seaborn: https://seaborn.pydata.org/generated/seaborn.lineplot.html#seaborn.lineplot